index.bs

```diff
@@ -1696,22 +1696,80 @@ Team Representative in an Interest Group</h5>
-	based on interest from the Members and Team.
-	The Team <em class="rfc2119">must</em> notify the Advisory Committee
-	when a charter for a new Working Group or Interest Group is in development.
-	This is intended to raise awareness,
-	even if no formal proposal is yet available.
-	Advisory Committee representatives <em class="rfc2119">may</em> provide
-	feedback on the <a href="#ACCommunication">Advisory Committee discussion list</a>
-	or via other designated channels.
-
-	W3C <em class="rfc2119">may</em> begin work
-	on a Working Group or Interest Group charter
-	at any time.
+	W3C creates [=charters=] for [=chartered groups=] based on interest from the [=Members=] and [=Team=].
+	[=Charters=] are formally approved through an [=AC Review=] and [=W3C Decision=] (see [[#CharterReview]]).
+	Prior to [=AC Review=],
+	<dfn>charter drafts</dfn> go through a public [=charter refinement=] phase
+	during which the [=charter=] receives [=wide review=]
+	and the [=Chartering Facilitator=] attempts to find [=consensus=] on the charter.
+	The [=Team=] <em class=rfc2119>may</em> send notice to the [=AC=]
+	when starting development of a new charter prior to the [=charter refinement=] phase.
+
+<h3 id="charter-initiation">
+Initiating Charter Refinement</h3>
+
+	Formal [=charter refinement=] (see below) is initiated
+	by the [=Team=] sending a [=charter review notice=]
+	to the [=Advisory Committee=],
+	to the public,
+	and, in the case of rechartering, to the affected [=Group=].
+
+	This <dfn>charter review notice</dfn> must include:
+	* A short summary of the proposal.
+	* The location of the [=charter draft=], which must be public.
+	* How to participate in the discussion of this [=charter draft=] and where to file issues.
+	* The expected duration of the [=charter refinement=] phase,
+		which must not be less than 28 days,
+		and should not be more than 6 months.
+	* Who the [=Chartering Facilitator=] is.
+
+	The [=Team=] is responsible for initiating [=charter refinement=]
+	at its discretion, in consideration of discussions with the community.
+	An [=Advisory Committee representative=] <em class=rfc2119>may</em> formally request
+	that the [=Team=] initiate [=charter refinement=].
+	The Team <em class=rfc2119>may</em> deny such a request
+	if it thinks the proposal is insufficiently mature or does not align with W3C's scope and mission.
+	This rejection is a [=Team Decision=] which can be <a href="#registering-objections">Formally Objected to</a>.
+
+<h3 id="charter-development" oldids="WGCharterDevelopment">
+Charter Refinement</h3>
+
+	During <dfn>charter refinement</dfn>,
+	the W3C community further develops the [=charter draft=]
+	with the goal of achieving [=consensus=] on the proposal.
+	The <dfn>Chartering Facilitator</dfn>
+	acts as [=Chair=] for the [=charter refinement=] process.
+
+	Note: The [=Chartering Facilitator=] is not necessarily (and often is not) the [=Chair=] of the [=chartered group|group=] being chartered.
```

Comment:
I suspect the Chartering Facilitator will usually be the Team Contact for the proposed group. Maybe call that out here?

Reply:
I don't think "usually the Team Contact for the proposed group" is an assumption that should be baked into Process at this time.

Reply:
It doesn't necessarily need to be the proposed team contact, but ideally should be someone impartial.

Reply:
Process CG resolved: Don't define who is the Chairing Facilitator, leave open-ended for now.